Elephone/Solider/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.Elephone.Solider"
+        android:versionCode="29"
+        android:versionName="10">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+		android:requiredSystemPropertyValue="+Elephone/Soldier/Soldier:8.0.0/O00623/20180823:user/release-keys*"
```

Comment:
Is it really useful to match to this precise vendor version?
Sounds counter-productive

Reply:
i used shortened name and send him the apk but he said it was not working

Reply:
Well that's wrong

Reply:
so should i use "+Solider" only?